code/modules/antagonists/changeling/changeling.dm

```diff
@@ -42,6 +44,18 @@
 	// wip stuff
 	var/static/list/all_powers = typecacheof(/obj/effect/proc_holder/changeling,TRUE)
 
+/datum/antagonist/changeling/New(list/changeling_list)
+	. = ..()
+	if(LAZYLEN(changeling_list))
+		competitive_objectives = (changeling_list.len > 1) //only give competivite objectives if we have more lings.
+	else //incase we're generated outside roundstart; ie create antag or latejoin antags
+		for(var/datum/antagonist/changeling/C in GLOB.antagonists)
+			if(!C.owner || C.owner == owner)
+				continue
+			if(C.was_absorbed) //make sure the other ling wasn't already killed by another one. only matters if the changeling that absorbed them was gibbed after.
+				continue
+			competitive_objectives = TRUE
```

Comment:
@AnturK So we have a choice here, I can add `team_mode = C.team_mode` to also fix latejoin changelings getting backstab goals when it's a changeling team based gamemode. Your call.

Reply:
Feels like something that should be set explicitly by admin_add / antagchance.